Source/Core/DolphinQt/Debugger/WatchWidget.cpp

```diff
@@ -250,9 +250,11 @@ void WatchWidget::OnClear()
 
 void WatchWidget::OnNewWatch()
 {
-  QString text = QInputDialog::getText(this, tr("Input"), tr("Enter address to watch:"));
+  const QString text =
+      QInputDialog::getText(this, tr("Input"), tr("Enter address to watch:"), QLineEdit::Normal,
+                            QString{}, nullptr, Qt::WindowCloseButtonHint);
```

Comment:
The first three new parameters are just default arguments.

Qt::WindowCloseButtonHint overrides the Qt::WindowFlags parameter with the only bit we want set, clearing the offending Qt::WindowContextHelpButtonHint.